src/code/code_800F7260.c

```diff
@@ -79,13 +79,13 @@ u8 gSfxChannelLayout = 0;
 
 u16 D_801333D0 = 0;
 
-Vec3f D_801333D4 = { 0.0f, 0.0f, 0.0f }; // default pos
+Vec3f gSfxPosScreenCenter = { 0.0f, 0.0f, 0.0f }; // default pos
```

Comment:
The current convention with position seems to be more like `[Qualifier][Space]Pos`, for example `targetScreenPosX`, `focusScreenPosX`, `effSpawnModelPos`

Now I don't know how much we care about it but we may as well be consistent and go with `gSfxCenterScreenPos`

But also this isn't screen cordinates, it's viewport coordinates (which go from -1,-1 to 1,1, where screen coordinates go from 0,0 to screen_width,screen_height), which we also call "projected" (`Actor_ProjectPos`, `Actor.projectedPos`)

So the better name would be `gSfxScreenCenterProjectedPos` / `gSfxScreenCenterViewportPos` 🤔
I would think that's very bad with how widely used this symbol is

Maybe we forget about saying what space it is and just call it `gSfxScreenCenterPos`, hoping it won't make someone think it's screen coordinates (highly likely confusion imo),
or name it something less useful like `gSfxDefaultPos` 😢 

another point for naming it `gSfxDefaultPos` would be this symbol is checked against in a way that makes it behave like a "default" I feel like:
https://github.com/zeldaret/oot/blob/d3b55beea3b2c981081712b50b6cb11178c56de5/src/code/code_800F7260.c#L373

and also consistency with the other "default" parameters

Reply:
I think trying to fit all of the information about what the values of the data mean is how we end up with really long verbose names. Its context dependent for sure, and sometimes it works out to do so, but in this case the longer names dont seem as beneficial to me personally

We can use default for all 3 and put comments next to the actual variables saying what the values mean. If someone is going to read those variable names and think to themselves "what are those defaults?", the first thing theyre going to do is go to their definition and read their values

Reply:
Alright, I've gone with:
`gSfxDefaultPos`
`gSfxDefaultFreqAndVolScale`
`gSfxDefaultReverb`

And added comments at their definition explaining what each of them defaults to